twindb_backup/destination/base_destination.py

```diff
@@ -174,3 +174,21 @@ def basename(self, filename):
         :return:
         """
         return filename.replace(self.remote_path + '/', '', 1)
+
+    def get_latest_backup(self):
+        """Get latest backup path"""
+        cur_status = self.status()
+        latest = None
+        max_finish = 0
+        for _, backups in cur_status.iteritems():
+            for key, backup in backups.iteritems():
+                try:
+                    if backup['backup_finished'] >= max_finish:
+                        max_finish = backup['backup_finished']
+                        latest = key
+                except KeyError:
+                    pass
+        if latest is None:
```

Comment:
If there are no backup_start and/or backup_finished and thus latest is None, sort available backup copy names and chose the latest one.